nix/sources.json

```diff
@@ -83,27 +83,29 @@
         "homepage": null,
         "owner": "dfinity",
         "repo": "motoko-base",
-        "rev": "45bd1abc8e81376d5e1c1dd6322e170e47555c57",
-        "sha256": "18i51cglq8131lyn6w2rjbhx32fbl7y62gga4cs10fdw2fmiw0d8",
+        "rev": "d0a814a3f93a3c168dacbeae2677f2b5d0995c02",
```

Comment:
Are we pulling in motoko-base directly now? That means we don't need to package base in motoko release.